TalZivelin/question3.js

```diff
@@ -0,0 +1,159 @@
+//3a.01 
+var objA = {a:1,b:0}
+var objB = {};
+
+function copy(objA){
+	
+	for ( var key in objA){
+		objB[key] = objA[key];
+	};
+  
+  return objB
+  
+};
+
+console.log(copy(objA));
+console.log(copy(objB));
+
+//3a.02
+
+function equal(objA,objB)	{	  
+	objB = objA;
+  if (objB == objA)	{
+		return true
+	}
+}
+
```

Comment:
This isn't quite right. on lin 21 you literally make objB point to the same thinng as objA. Meaning instead of testing if B or A are the same, you MAKE them the same. so this test will always pass. You want to loop through each jey in the object and see if they are === to eachother. 

example: 
```
function equal(objA,objB) {
    var key; // will be set to each prop name of each object
    var match; // will be true or false for each prop comparison

    // check that B has everything A does:
    for (key in objA) {
        match = ((key in objB) &&
                 (objA[key] === objB[key])); //Note: NaN will never match!
        if (!match)
            return false;
    }
    // check that A has everything B does:
    for (key in objB) {
        match = ((key in objA) &&
                 (objA[key] === objB[key]));
        if (!match)
            return false;
    }
    return true;
}
```